src/components/PokemonDexInfo.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from "react"
+
+export default function PokemonDexInfo({ id }) {
+  const [pokemonDexInfo, setPokemonDexInfo] = useState({})
+  const [isLoaded, setIsLoaded] = useState(false)
+  const [error, setError] = useState(null)
+  
+  useEffect(() => {
+    fetch("https://pokeapi.co/api/v2/pokemon-species/"+id)
```

Comment:
Suggestions, most of the linters advise to use template literals instead of concatenation, template literals is a new feature of js that allows you to `https://pokeapi.co/api/v2/pokemon-species/${id}` do it like this surrounding the js in `${}` this